js/main.js

```diff
@@ -515,8 +590,10 @@ const MM = (function () {
 		 * Update the dom for a specific module.
 		 * @param {Module} module The module that needs an update.
 		 * @param {number} [speed] The number of microseconds for the animation.
+		 * @param {string} [animateOut] AnimateCss animation name before hidden
+		 * @param {string} [animateIn] AnimateCss animation name on show
 		 */
-		updateDom: function (module, speed) {
+		updateDom: function (module, speed, animateOut, animateIn) {
```

Comment:
in a perfect world we would refactor the sighanture to 
(module, {options: {
 speed,
 animate: {
  out,
  in
}) and have a fallback to the old signature. what do you think?

Reply:
thx for the refactor